Ice36 release

jburel opened the pull request:
Install ice 3.6 on all os
git is installed but will be removed when we can use omego.
the `step04_all_omero.sh` file also need to be modified when the openmicroscopy PR is merged.

This is not ready to be merged

jburel committed:
Install ice36 from source on centos7

jburel committed:
Add ice36 env

jburel committed:
Install git to install from source

jburel committed:
Build from source

jburel committed:
Add ice3.6 support for ubuntu

jburel committed:
install git on ubuntu

jburel committed:
Build ice3.6 on centos6

jburel committed:
Pass the ice version as a parameter

jburel committed:
Build from source if ice36

jburel committed:
Install git.

jburel committed:
Set the variables.

jburel committed:
Add missing dependencies.

jburel committed:
Build ice from source.

jburel committed:
Set ICE_HOME.

jburel committed:
Add ice36 to the list of supported values

jburel committed:
Fix doc

jburel committed:
Use Ice 3.6.2 on centos6

jburel committed:
Fix ice version check

jburel committed:
Install ice release version on centos7

jburel committed:
Add dependency to manage Berkeley DB

jburel committed:
Remove code no longer needed.

jburel committed:
Build from source using release version.

jburel committed:
Remove code no longer needed.

jburel committed:
Remove commented out line

jburel committed:
Bump version

jburel committed:
Update export

jburel committed:
Install zeroc-ice using pip2.7

jburel committed:
Re-organize files to install in virtualenv

jburel committed:
Build using the release source

jburel committed:
Improve installation of ice 3.6 centos6 py27 ius.

jburel committed:
Improve ice installation on centos6 py27 scl

jburel committed:
Improve installation on ubuntu/debian

the zeroc link is now working.

jburel committed:
Remove env file no longer needed

jburel committed:
Update readme

jburel committed:
Fix the doc

jburel committed:
use bin/bash instead of bin/sh

atarkowska commented on linux/step01_centos7_ice_deps.sh:
```diff
@@ -15,4 +15,18 @@ elif [ "$ICEVER" = "ice35-devel" ]; then
+	wget https://zeroc.com/download/rpm/zeroc-ice-el7.repo
+
+	#for omero, to remove when we do not install from git
+	yum -y install git
```
is there any reason for installing get as a part of ice dependencies? why not do it in https://github.com/ome/omero-install/pull/102/files#diff-49a05e9c183b6b84aa966891a004eb37R30

jburel replied:
I put it in ice 36 section b/c of omero installation, It will be removed as soon as I use label as indicated in the description
It is not done in step04_omero since all os scripts now use it.

atarkowska commented on linux/test/test_services.sh:
```diff
@@ -42,14 +42,14 @@ docker exec -it $CNAME /bin/bash -c 'd=10; \
 
 #check OMERO.server service status
-docker exec -it $CNAME /bin/bash -c "service omero status -l"
+docker exec -it $CNAME /bin/bash -c "systemctl status omero.service -l"
```
if https://github.com/ome/ome-docker/pull/36/files is in, the above can be reverted

jburel replied:
Reverted the changes in test waiting for the upstream change

manics commented:
Is there an ice36 CI build? omego should already work with it, though you might need to add a labels argument.

jburel commented:
@manics: we don't have one yet, as soon as it is working I will revert and use labels.
That's the plan cf. description

manics commented:
@jburel Thanks, I misunderstood the description- I thought you meant changes were needed to omego

jburel committed:
Install initscripts so test script can be used for ice 3.5/3.6

jburel commented:
FYI: Debian currently not passing. Looking into it.

jburel committed:
Build from source on debian

jburel commented:
due to constant failure on debian, building from source for now

jburel committed:
Refactor to add option to generate all walkthrough

jburel committed:
Fix comment

jburel committed:
Fix comment

jburel commented:
all, expect centos7_apache, are green cf https://travis-ci.org/ome/omero-install/builds/123329750.
Reverting commit.

jburel committed:
Fix omero installation section.

jburel committed:
Improve Apache section.

jburel committed:
Add new markers

jburel committed:
Fix typo

jburel committed:
Remove cento6s from the list of generated walkthrough

It is still possible to generate the walkthrough for centos6
but it won't be generated when the ALL flag is used.

jburel committed:
Install omero using ice 3.6 build

jburel committed:
No longer instal git.

jburel committed:
No longer use omego to install release version

jburel committed:
Allow to run script for other locations

jburel committed:
Point to breaking ice36 zip

jburel committed:
Add markers to add ice 3.6 to walkthrough

jburel committed:
Remove ice3.6 section

jburel committed:
Add markers for doc

jburel commented:
note that the omero set-up will be adjusted prior to release.
Currently pointing a build on breaking
